### PR TITLE
 Adding API1600 support for Scope resource

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
Adding API1600 support for Scope resource

### Check List
- [ ] New functionality includes testing.
  - [x] All tests pass (`$ rake test`).
- [ ] New functionality has been documented in the README if applicable.
  - [x] New functionality has been thoroughly documented in the examples (please include helpful comments).
- [x] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG.

### CODE REVIEW

File: examples/shared_samples/scope.rb [hunk 1]
```diff
@@ -29,6 +37,15 @@
 # api_version = 500 & variant = Synergy to OneviewSDK::API500::C7000::Scope
 # api_version = 600 & variant = C7000 to OneviewSDK::API600::C7000::Scope
 # api_version = 600 & variant = Synergy to OneviewSDK::API600::C7000::Scope
+# api_version = 800 & variant = C7000 to OneviewSDK::API800::C7000::Scope
+# api_version = 800 & variant = Synergy to OneviewSDK::API800::C7000::Scope
```
Comment (chebroluharika) [inline examples/shared_samples/scope.rb:41]: It should be OneviewSDK::API800::Synergy::Scope

File: examples/shared_samples/scope.rb [hunk 2]
```diff
@@ -29,6 +37,15 @@
...
+# api_version = 1000 & variant = C7000 to OneviewSDK::API1000::C7000::Scope
+# api_version = 1000 & variant = Synergy to OneviewSDK::API1000::C7000::Scope
```
Comment (chebroluharika) [inline examples/shared_samples/scope.rb:43]: same as above

File: examples/shared_samples/scope.rb [hunk 3]
```diff
@@ -29,6 +37,15 @@
...
+# api_version = 1200 & variant = C7000 to OneviewSDK::API1200::C7000::Scope
+# api_version = 1200 & variant = Synergy to OneviewSDK::API1200::C7000::Scope
```
Comment (chebroluharika) [inline examples/shared_samples/scope.rb:45]: same as above

File: examples/shared_samples/scope.rb (hunk 1 above)
Comment (yuvirani) [reply examples/shared_samples/scope.rb]: Updated as mentioned

File: examples/shared_samples/scope.rb (hunk 2 above)
Comment (yuvirani) [reply examples/shared_samples/scope.rb]: Done

File: CHANGELOG.md [hunk 4]
```diff
@@ -19,7 +19,6 @@ This release extends support of the SDK to OneView REST API version 1600 (OneVie
 - Logical Interconnect
 - Logical Interconnect Group
 - OS Deployment Plan
-- Server Certificate
```
Comment (AsisBagga) [inline CHANGELOG.md:22]: Not sure why Server Certificate was removed, 
Need to add Scope in the list.

Comment (yuvirani) [reply CHANGELOG.md:22]: Resolved Merge conflicts

**Pull request merged**